recipe/meta.yaml

```diff
@@ -1,28 +1,29 @@
 {% set name = "coffea" %}
 {% set version = "2024.10.0" %}
+{% set python_min = "3.8" %}
```

Comment:
This will now ensure that there are versions that work with the lower bound of the `requires-python` defined in the PyPI package. If the PyPI `requires-python` changes this workflow will fail (which is good) and it will then require bumping this.